game/recastnavigation_bin.ksy

```diff
@@ -0,0 +1,283 @@
+meta:
+  id: recastnavigation_bin
+  file-extension: bin
+  license: CC0-1.0
+  endian: le
+doc: this spec can be used to parse recastnavigation binary files
+doc-ref: https://masagroup.github.io/recastdetour
+seq:
+  - id: header
+    type: navmesh_set_header
+  - id: tiles
+    type: navmesh_tile
+    repeat: expr
+    repeat-expr: header.num_tiles
+types:
+  navmesh_set_header:
+    seq:
+      - id: magic
+        contents: "TESM"
+      - id: version
+        type: s4
+      - id: num_tiles
+        type: s4
+      - id: params
+        type: dt_navmesh_params
+  dt_navmesh_params:
```

Comment:
Same thing with this type — it can be easily part of the `header` or top-level type.